test/docfx.Tests/JsonConverterTest.cs

```diff
@@ -81,6 +82,30 @@ public void TestFileMappingItemOutputShouldContainSrcOnly()
             Assert.Equal("{\"files\":[\"file1\"],\"src\":\"folder1\"}", result);
         }
 
+        [Fact]
+        [Trait("Related", "docfx")]
+        public void TestManifestItemCollectionConverterCouldSerializeAndDeserialize()
+        {
+            var manifest = new Manifest();
+            ManifestItem manifestItemA = new ManifestItem
+            {
+                SourceRelativePath = "a"
```

Comment:
Yeah, it is better!

---
In reply to: [126617067](https://github.com/dotnet/docfx/pull/1847#discussion_r126617067) [](ancestors = 126617067)